src/components/Question.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import Button from './Button';
+
+const Question = (question) => {
+  const [selectedOption, setSelectedOption] = useState(null);
+  const [isCorrect, setIsCorrect] = useState(null);
+
+  const handleOptionChange = (event) => {
+    const correctAnswer = event === question.rightAnswer;
+
+    setSelectedOption(event);
+    setIsCorrect(correctAnswer);
+  };
+
+  return (
+    <div key={question.id} className="py-7">
```

Comment:
please check how keys are used and passed

Reply:
fixed